packages/auth/package.json

```diff
@@ -12,6 +12,7 @@
     "build": "pnpm build:deps && pnpm clean && tsc --build tsconfig.json && pnpm copy-files",
     "clean": "rm -fr dist/",
     "test": "jest --passWithNoTests --maxWorkers=50%",
+    "test:cov": "pnpm test -- --coverage --changedSince=main",
```

Comment:
Are the two `--` after `test` intentional since nothing is following them immediately?

Reply:
yeah its weird, but that will pass the `--coverage` and `--changedSince` args to the `test` command. Without it it gets interpreted as arguments for the `pnpm` command.

Reply:
The more you know... Thank you!